app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -82,15 +82,13 @@ fun View.toggleChevron(
     animate().rotation(angle).setDuration(duration).start()
 }
 
-fun View.setMargins(left: Int? = null, top: Int? = null, right: Int? = null, bottom: Int? = null) {
+fun View.setMargins(start: Int? = null, top: Int? = null, end: Int? = null, bottom: Int? = null) {
     if (layoutParams is ViewGroup.MarginLayoutParams) {
         (layoutParams as ViewGroup.MarginLayoutParams).apply {
-            setMargins(
-                left ?: leftMargin,
-                top ?: topMargin,
-                right ?: rightMargin,
-                bottom ?: bottomMargin,
-            )
+            start?.let { marginStart = it }
+            top?.let { topMargin = it }
+            end?.let { marginEnd = it }
+            bottom?.let { bottomMargin = it }
         }
         requestLayout()
```

Comment:
Should be put in Core, as used by kDrive too
https://github.com/Infomaniak/android-kDrive/blob/4e7d87d07032c6c852218b6b7a2f28d7d5264d6a/app/src/main/java/com/infomaniak/drive/utils/Extensions.kt#L180